api/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Octopus API documentation
+
+This directory contains the files needed to deploy a static [documentation page](https://jiscsd.github.io/octopus/) for the octopus API.
+
+The majority of the files are taken straight from [Swagger UI](https://swagger.io/tools/swagger-ui/), and our own OpenAPI specification is also included (`api.yml`).
```

Comment:
Might want to update this now you're using unpkg

Reply:
Good point! Now done